tutor/resources/styles/mixins/performance-forecast.scss

```diff
@@ -58,12 +58,17 @@
 
 @mixin performance-forecast-key() {
   display: flex;
-  justify-content: space-evenly;
-  min-width: 280px;
-  margin-top: 0.5rem;
+  /* https://coryrylan.com/blog/css-gap-space-with-flexbox */
+  --gap-top: 5px;
+  --gap-bottom: 12px;
+  margin: calc(-1 * var(--gap-top)) 0 0 calc(-1 * var(--gap-bottom));
+  width: calc(100% + var(--gap));
+  margin-top: 1rem;
   .item {
     display: flex;
     align-items: center;
+    align-items: flex-start;
+    margin: var(--gap-top) 0 0 var(--gap-bottom);
     @include performance-forecast-colors();
     .progress-bar {
       width: 10px;
```

Comment:
did this creep over from another PR?

Reply:
I think so. The changes are in the main branch already.

Reply:
Weird, if you compare them again. It does not show these changes. I will close this PR and create a new one, just in case.

Reply:
mmm. I can't close this PR. I think we should be good at merging this.

Reply:
yep. it merged fine to the main branch.